dev-tools/notice/overrides.json

```diff
@@ -1,4 +1,6 @@
 {"name": "github.com/elastic/elastic-agent-client/v7", "licenceType": "Elastic"}
+{"name": "github.com/AndersonQ/elastic-agent-client/v7", "licenceType": "Elastic"}
```

Comment:
are you referencing own repos?

Reply:
Yes, only until https://github.com/elastic/elastic-agent-client/pull/56 is merged.  After that we can remove this line.

Reply:
Removed.